### PR TITLE
Updated readme.md file, created user entities and account entities.

### PR DESCRIPTION
**Pull request by serhatbsts**
Updated readme.md file, created user entities and account entities.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,131 @@
-https://github.com/serhatbsts/ComputerScienceStudentsSummerHomeWork
+# Computer Science Students Summer Home-Work
+
+If you are a Computer Science student and want to improve your programming skills, you can start this project. This project will help you to improve your programming skills. You can do this project in your summer holiday.
```
Comment (karaelf33) [inline README.md:3]: please update Read me information only related account manager project.
This all info is not related in current project

File: src/main/java/com/Project/accountManager/entities/Account.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    int accountNumber;
+
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JsonIgnore
+    User user;
+
+    public void setId(Long id) {
+        this.id = id;
```
Comment (karaelf33) [inline src/main/java/com/Project/accountManager/entities/Account.java:25]: @Data annotation is already contains getter and setter.
Either delete setters and only use @Data annotation or delete @data annotation and use setters manually

File: src/main/java/com/Project/accountManager/entities/User.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package com.Project.accountManager.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Table(name="user")
```
Comment (karaelf33) [inline src/main/java/com/Project/accountManager/entities/User.java:9]: @serhatbsts , Mysql contains some reserved keywords. that's why you can't use some keywords "user" "account" etc...
Please read this document ->  https://dev.mysql.com/doc/refman/8.0/en/keywords.html
Screenshot 2024-05-24 at 06 18 49
Screenshot 2024-05-24 at 06 19 06

File: src/main/java/com/Project/accountManager/entities/Account.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
+package com.Project.accountManager.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+@Entity
+@Table(name="account")
```
Comment (karaelf33) [inline src/main/java/com/Project/accountManager/entities/Account.java:10]: Same like here https://github.com/serhatbsts/accountManager/pull/2/files#r1612716592

File: src/main/java/com/Project/accountManager/entities/User.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+@Data
+public class User {
+    @Id
+    private Long id;
+    String name;
+    String surName;
+    int password;
+    public void setId(Long id) {
```
Comment (karaelf33) [inline src/main/java/com/Project/accountManager/entities/User.java:17]: Same like here ->https://github.com/serhatbsts/accountManager/pull/2/files#r1612713627

File: README.md (hunk 1 above)
Comment (serhatbsts) [reply README.md:3]: OK, necessary corrections will be made.

File: src/main/java/com/Project/accountManager/entities/Account.java (hunk 2 above)
Comment (serhatbsts) [reply src/main/java/com/Project/accountManager/entities/Account.java:25]: OK, necessary corrections will be made.

File: src/main/java/com/Project/accountManager/entities/User.java (hunk 3 above)
Comment (serhatbsts) [reply src/main/java/com/Project/accountManager/entities/User.java:9]: OK, necessary corrections will be made.

File: src/main/java/com/Project/accountManager/entities/Account.java (hunk 4 above)
Comment (serhatbsts) [reply src/main/java/com/Project/accountManager/entities/Account.java:10]: OK, necessary corrections will be made.

File: src/main/java/com/Project/accountManager/entities/User.java (hunk 5 above)
Comment (serhatbsts) [reply src/main/java/com/Project/accountManager/entities/User.java:17]: OK, necessary corrections will be made.
